src/apps/settings/i18n/translations/it.yml

```diff
@@ -1,7 +1,7 @@
 sides:
-  top: Superiore
+  top: Sopra
   left: Sinistra
-  bottom: Metter il fondo a
```

Comment:
change this lol

Reply:
wdym? i changed that already

Reply:
yeah I was recently waking up hahaha sorry